p2p/server.go

```diff
@@ -670,32 +637,31 @@ type dialer interface {
 	isStatic(*enode.Node) bool
 }
 
-type valNodeInfo struct {
-	atValRemoveSetStatic  bool
-	atValRemoveSetTrusted bool
-	remoteEnodeURL        string
-}
-
 func (srv *Server) run(dialstate dialer) {
 	srv.log.Info("Started P2P networking", "self", srv.localnode.Node())
 	defer srv.loopWG.Done()
 	defer srv.nodedb.Close()
 
 	var (
-		peers                = make(map[enode.ID]*Peer)
-		inboundCount         = 0
-		trusted              = make(map[enode.ID]bool, len(srv.TrustedNodes))
-		taskdone             = make(chan task, maxActiveDialTasks)
-		runningTasks         []task
-		queuedTasks          []task // tasks that can't run yet
-		valNodes             = make(map[enode.ID]*valNodeInfo)
-		numConnectedValPeers = 0
-		numInboundValPeers   = 0
+		static       = make(map[enode.ID]map[string]bool, len(srv.StaticNodes))
```

Comment:
I might not 100% understand the difference between a static peer label and a trusted peer label, so excuse me if this is a dumb question, but could we not just keep a single map of nodes => labels? It seems to me that `sentry` and `validator` labels basically act as a strict super set to at least `trusted` (and also `static` if dialed). If that is the case, couldn't we simplify by using the labels directly as the medium for determining whether peers should be marked as static/trusted?